src/Dapr.Client/DaprClientBuilder.cs

```diff
@@ -57,6 +57,7 @@ public DaprClientBuilder()
         // property exposed for testing purposes
         internal GrpcChannelOptions GrpcChannelOptions { get; private set; }
         internal string DaprApiToken { get; private set; }
+        internal TimeSpan Timeout { get; private set; } // in milliseconds
```

Comment:
Nit: remove comment?  (No longer in milliseconds?)

Reply:
Good catch, thanks. Done!